kip2016/my_store2
# Ft shopping cart page 161173284

**What does this PR do?**
User should be able to add products to the cart.

**Description of Task to be completed?**
The user should be able to add products to the cart in the  system.

**How should this be tested?**
Clone the repository to you local machine then access the page using your local URL to your a browser. Try to add to cart.

Background context you want to provide?
It is a requirement with the project guidelines that the Login credentials should be validated.
